<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>535</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>535</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>535</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Qute MTG Stats</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>535</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="dataTab">
     <attribute name="title">
      <string>Raw Data</string>
     </attribute>
     <widget class="QTextEdit" name="statsPaste">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>795</width>
        <height>401</height>
       </rect>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="phraseButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>781</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Phrase Data and Generate Stats</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Paste Data Below:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="formatTab">
     <attribute name="title">
      <string>Stats by Format</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="eventTab">
     <attribute name="title">
      <string>Stats by Event Type</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="opponentTab">
     <attribute name="title">
      <string>Stats by Opponent</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>795</width>
        <height>501</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Opponent</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Wins</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Losses</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Draws</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Total</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <action name="actionMenu_Item">
   <property name="text">
    <string>Menu Item</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
